sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -545,43 +543,46 @@ def test_string_tests(self):
 
   def test_datetime_tests(self):
     # TODO(BEAM-10721)
-    datetimelike_result = doctests.testmod(
-        pd.core.arrays.datetimelike,
+    indexes_accessors_result = doctests.testmod(
+        pd.core.indexes.accessors,
         use_beam=False,
         skip={
-            'pandas.core.arrays.datetimelike.AttributesMixin._unbox_scalar': [
+            'pandas.core.indexes.accessors.TimedeltaProperties': [
+                # Seems like an upstream bug. The property is 'second'
+                'seconds_series.dt.seconds'
+            ],
+
+            # Test data creation fails for these:
```

Comment:
Why?

Reply:
Thanks for asking about this. This is an issue with the way we shim constant values in the doctest framework. I filed BEAM-12530 for this and linked it here.

Also I had opened https://github.com/apache/beam/pull/13457 for this a while ago and completely forgot about it. Probably worth dusting that off for this.